### PR TITLE
fix: import next.config.js in Windows

### PR DESCRIPTION
**Pull request by luwes**
related #151

it's fixed, the issue seems to be that Windows path.resolve() behaves differently than Unix.
it did require a cwd() argument while Unix didn't it seemed.

`path.resolve(cwd(), file);`

I also added `pathToFileURL` to make sure it's a valid URL for JS dynamic import() because it expects an URL, not a path.
https://github.com/nodejs/node/issues/34765

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// This file is used in the tests to mock the next.config.js file!!!
+// It's not actually used by the library, it's just a mock file for testing purposes.
```
Comment (heff) [inline next.config.js:2]: Suggestion: Put this file in /tests if it doesn't need to live in the root

**Review by heff — APPROVED** [review]
lgtm, one suggestion

File: next.config.js (hunk 1 above)
Comment (luwes) [reply next.config.js:2]: yep, I tried that but quickly moved it to the root to power through.
I'll try with Christian's suggestion of using a env variable which we can manually set for running tests

Comment (heff) [reply next.config.js:2]: Cool. 👍

**Pull request merged**